mosdepth/Dockerfile

```diff
@@ -33,4 +33,4 @@ LABEL snakemake-wrapper-utils=0.3.1
 # to /usr/local to make it possible
 # running the softwares without activating
 # any conda env
-COPY --from=build /venv /usr/local
```

Comment:
I know this is a correct change but it will remake the docker. Maybe leave what is working as is?